src/HuffConfig.sol

```diff
@@ -196,6 +205,7 @@ contract HuffConfig {
 
         /// @notice deploy the bytecode with the create instruction
         address deployedAddress;
+        if (should_broadcast) vm.broadcast();
```

Comment:
Testing via a foundry script, the `CREATE` instruction doesn't seem to include the bytecode of the Huff contract. Will look into why this is tomorrow:

```
  {
      "hash": null,
      "transactionType": "CREATE",
      "contractName": null,
      "contractAddress": "0x7c276dcaab99bd16163c1bcce671cad6a1ec0945",
      "function": null,
      "arguments": null,
      "transaction": {
        "type": "0x00",
        "from": "0x00a329c0648769a73afac7f9381e08fb43dbea72",
        "gas": "0x10d24",
        "value": "0x0",
        "data": "0x", // <---- No data was included for the call
        "nonce": "0x4"
      }
    }
```

Reply:
Not sure why it's not showing up there, but I've tested this branch in one of my repos [here](https://github.com/ncitron/cell-huff/blob/master/script/DeployCell.s.sol) and it deploys fine.

Reply:
> Not sure why it's not showing up there, but I've tested this branch in one of my repos [here](https://github.com/ncitron/cell-huff/blob/master/script/DeployCell.s.sol) and it deploys fine.

Ah, my script had a bug in the path to the Huff contract passed to `.broadcast`, whoops.